### DISCUSSION TITLE
Add map to your website

### DISCUSSION THREAD

**Discussion by amarangunic** [earthlab-education/Earth-Analytics-AY24#300] [Q&A]
Hello, I am having trouble to use the codespace and use the readme file. 
The error indicates:
Error al cargar la vista web: Error: Could not register service worker: SecurityError: Failed to register a ServiceWorker: The provided scriptURL ('https://03atri7pchng9kf0lloi1q4fj74mbf7de47f5f58scrjs1o1lfb7.assets.github.dev/stable/4849ca9bdf9666755eb463db297b69e5385090e3/out/vs/workbench/contrib/webview/browser/pre/service-worker.js?v=4&vscode-resource-base-authority=vscode-resource.vscode-cdn.net&remoteAuthority=codespaces+jubilant-parakeet-4qxgv4xqvrvc7g67') violates the Content Security Policy..

Thanks

**Comment by amarangunic** [comment]
And also it indicates this: 
Could not initialize webview: Error: Could not register service worker: SecurityError: Failed to register a ServiceWorker: The provided scriptURL ('https://1trg6uld3j48u5m6924067qom72h1s23agkvtu0ncfkdcf5a5joe.assets.github.dev/stable/4849ca9bdf9666755eb463db297b69e5385090e3/out/vs/workbench/contrib/webview/browser/pre/service-worker.js?v=4&vscode-resource-base-authority=vscode-resource.vscode-cdn.net&remoteAuthority=codespaces+miniature-fishstick-gxg5r9gxp9gh954g') violates the Content Security Policy..}

**Comment by eculler** [comment]
Hi @amarangunic, sorry this isn't working for you! It looks very frustrating.

Taking a closer look at your error message, I believe you may be running into some kind of content blocker. It could be your web browser, your computer, your network, or the country you're in. Let's try to meet to troubleshoot this. If you want to try some things yourself first, I'd recommend:

1. Try a different web browser. We've had success with Chrome and Firefox, and mostly run into problems with Safari. However, it could also be some settings that are on one browser but not another.
2. If you use a content or ad blocker, disable it while working on Codespaces. We have found that unfortunately whitelisting the domain doesn't usually work -- the blocker must be turned off entirely while you use Codespaces.
3. If you are trying on a work, school, or library network (or you are using a VPN to an institutional network), try on your home network if available.
4. If you are on a computer from work, try on another computer if you have one available.
5. If you know that the country you're in restricts some parts of the internet, they may be restricting Codespaces and there's not a lot we can do to legally get around that. We can, however, help you get software installed locally.

If none of that works, let's go ahead and meet, and we can get you set up with an installation of the software you will need on your computer. If you're on a work computer, you may need to get your IT department to do the install, so this also has a higher chance of success on a personal computer.

We can also try the [Google Colab](https://colab.research.google.com/) platform. It is less customizable than GitHub Codespaces, but more robust across different systems and network restrictions. You can even get notebooks from GitHub, and commit them back when you're done, although it's a little clunkier than Codespaces. We have, however, had some challenges getting holoviews plots like your interactive map to display in Colab, so we might need to find some workarounds for that.
